node/caching-compiler.js

```diff
@@ -70,7 +71,7 @@ function readWithCache(info, content) {
   };
 
   if (reify && reify.parser) {
-    compileOptions.parse = require(reify.parser).parse;
+    compileOptions.parse = utils.require(reify.parser).parse;
```

Comment:
Ah each require is per instance of the module so it could effect resolution. There is no `module.require` in webpack transformed code so I might have to move the transform per file (that's how I had it before I tried to consolidate).